src/main.rs

```diff
@@ -75,19 +75,15 @@ fn main() -> anyhow::Result<()> {
         std::process::exit(0);
     }
 
-    match args.command {
-        Some(SemverChecksCommands::CheckRelease(args)) => {
-            let check: cargo_semver_checks::Check = args.into();
-            let report = check.check_release()?;
-            if report.success() {
-                std::process::exit(0)
-            } else {
-                std::process::exit(1);
-            }
-        }
-        None => {
-            anyhow::bail!("subcommand required");
-        }
+    let check: cargo_semver_checks::Check = match args.command {
+        Some(SemverChecksCommands::CheckRelease(args)) => args.into(),
+        None => args.check_release.into(),
+    };
+    let report = check.check_release()?;
+    if report.success() {
+        std::process::exit(0)
```

Comment:
```suggestion
        std::process::exit(0);
```
I just noticed the other branch has a `;` and this one didn't.

This was already present before you made any changes so you did the right thing by leaving it as-is. I'm familiar with the context of this code and the `;` vs not doesn't make a difference, but it's nice to be consistent :)